keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -697,8 +701,9 @@ it will be safe to disable the dual-write mode.
 | 1.31     | Beta off | Alpha off   |
 | 1.32     | Beta on  | Alpha off   |
 | 1.33     | GA on    | Beta off   |
-| 1.34     | GA (there are no bitmaps running) | GA on (also delete old bitmap)|
-| 1.35     | remove feature gate | remove feature gate  |
+| 1.34     | GA (there are no bitmaps running) | GA (also delete old bitmap)|
+| 1.36     | remove feature gate | GA  |
```

Comment:
Small nit: just for visibility I'd probably add the row for 1.35, some folks might not catch that it's being skipped :wink: